colcon-test/README.md

```diff
@@ -0,0 +1,83 @@
+# colcon-test
+
+This action runs `colcon test` with label.
```

Comment:
```suggestion
This action runs `colcon test` with labels.
```

Reply:
This action accepts only one label as input. Therefore, the plural form is not appropriate.

Reply:
Then, let's add `a`. Also, the default value `.*` seems strange to me.

Reply:
```suggestion
This action runs `colcon test` with a label.
```

Reply:
But why can't we use multiple tags here? Are there some technical reasons?

Reply:
When using the CodeCov action, the label name and coverage result must be specified for each label. As a result, we will be executing as many CodeCov actions as there are labels.
In this case, we may want to use a matrix workflow. To use the matrix, I decided to use only one tag.

Reply:
I see, thank you.

Is it impossible to use different tags for `colcon test` and `CodeCov`?
I think it can support flexible use cases.

Reply:
> Is it impossible to use different tags for colcon test and CodeCov?
I think it can support flexible use cases.

It is possible. If you want to execute gtest and launch_test at once, you can specify `label: "gtest|launch_test"`. And send it to CodeCov with `tags: gtest_and_launch_test` argument.